src/file_organizer/services/audio/transcriber.py

```
@@ -155,13 +163,17 @@ def _detect_device(self, device: str) -> str:
-        """Lazy load the Whisper model."""
+        """Load the Whisper model, raising ImportError if faster-whisper is not installed."""
         if self._model is not None:
             return self._model
 
-        try:
-            from faster_whisper import WhisperModel  # pyre-ignore[21]  # pragma: no cover
+        if not _FASTER_WHISPER_AVAILABLE:
+            raise ImportError(
+                "faster-whisper is required for audio transcription. "
+                "Install it with: pip install faster-whisper"
+            )
```

Ruff (0.15.9)

[warning] Avoid specifying long messages outside the exception class

(TRY003)